post-publish-processor/src/main/scala/org/sunbird/job/postpublish/helpers/DialHelper.scala

```
@@ -0,0 +1,127 @@
+        .asInstanceOf[util.Map[String, AnyRef]].getOrDefault("reservedDialcodes", new util.HashMap[String, Integer]())
+        .asInstanceOf[util.Map[String, Integer]]
+      reservedDialcodes
+    } else throw new Exception(s"Couldn't reserve dialcodes for identifier: $identifier")
+  }
+
+  def updateDIALToObject(identifier: String, dialCode: String)(implicit neo4JUtil: Neo4JUtil) = {
+    neo4JUtil.updateNodeProperty(identifier, "dialcodes", (s"""["${dialCode}"]"""))
+    logger.info(s"Added Reserved Dialcode to node.")
+  }
+
+  def fetchExistingDialcodes(edata: java.util.Map[String, AnyRef]): util.List[String] = {
+    edata.getOrDefault("dialcodes", new util.ArrayList()).asInstanceOf[util.List[String]]
+  }
+
+  def validatePrimaryCategory(edata: java.util.Map[String, AnyRef])(implicit config: PostPublishProcessorConfig) = {
+    val primaryCategory = edata.get("primaryCategory").asInstanceOf[String]
+    config.primaryCategories.contains(primaryCategory)
+  }
+
+  def validateQR(dialcode: String)(implicit extConfig: ExtDataConfig, cassandraUtil: CassandraUtil): Boolean = {
+    getQRImageRecord(dialcode) match {
+      case Some(url: String) => true
+      case _ => false
+    }
+  }
+
+  def getQRImageRecord(dialcode: String)(implicit extConfig: ExtDataConfig, cassandraUtil: CassandraUtil): Option[String] = {
+    if (dialcode.isEmpty) throw new Exception("Invalid dialcode to read")
+    val fileName = s"0_$dialcode"
+    val query = s"select url from ${extConfig.keyspace}.${extConfig.table} where filename = '$fileName';"
+    try {
+      val result = cassandraUtil.findOne(query)
+      if (result == null) None else Some(result.getString("url"))
+    } catch {
+      case e: Exception => logger.error("There was an issue while fetching the qr image url from table")
+        throw new Exception("There was an issue while fetching the qr image url from table")
+    }
```

Remove this try-catch also. We are suppressing the actual exception here.